ci/cloudbuild/notifiers/alerts/README.md

```diff
@@ -0,0 +1,69 @@
+# Send alerts when CI builds fail
+
+This notifier listens for failed builds (ignoring PR and manual builds) and
+sends alert chats to the webhook URL specified by the `GCB_BUILD_ALERT_WEBHOOK`
```

Comment:
"alert chats" sounds a bit awkward.  I think we should make it explicit that the webhook is for Google Chat, I mean, you could not send the same format to Slack I presume?

Reply:
You're correct. Fixed.